src/Dialogs/Dialogs.vala

```diff
@@ -604,9 +604,13 @@ public bool report_manifest (ImportManifest manifest, bool show_dest_id,
 }
 
 internal void save_import_results (Gtk.Window? chooser_dialog_parent, string results_log) {
-    Gtk.FileChooserDialog chooser_dialog = new Gtk.FileChooserDialog (
-        ImportUI.SAVE_RESULTS_FILE_CHOOSER_TITLE, chooser_dialog_parent, Gtk.FileChooserAction.SAVE,
-        (_ ("_Cancel")), Gtk.ResponseType.CANCEL, (_ ("_Save")), Gtk.ResponseType.ACCEPT, null);
+    var chooser_dialog = new Gtk.FileChooserNative (
+        ImportUI.SAVE_RESULTS_FILE_CHOOSER_TITLE,
+        chooser_dialog_parent,
+        Gtk.FileChooserAction.SAVE,
+        _("_Save"),
+        _("_Cancel")
```

Comment:
Is it normal to also include a "_" in the text itself?

Instance without:
https://github.com/elementary/photos/pull/490/files#diff-233c585695efb8f696a2531fc779b1d6R619

Instance with:
https://github.com/elementary/photos/pull/489/files#diff-555c8c93465c46c845291e9d468aa2cfR93

Reply:
I see usually no "_", but other instances with "_":
https://github.com/elementary/photos/blob/master/src/Dialogs/Dialogs.vala#L93
https://github.com/elementary/videos/blob/master/src/Window.vala#L389
https://github.com/elementary/code/blob/master/src/MainWindow.vala#L781
https://github.com/elementary/music/blob/master/src/Widgets/AlbumListGrid.vala#L251
https://github.com/elementary/videos/blob/master/src/Widgets/Library/LibraryItem.vala#L171

Reply:
It's for button mnemonics, for faster keyboard navigation. So you can press like "alt c" to cancel. We _probably_ should use them everywhere